solutions/Fully configurable/variables.tf

```diff
@@ -0,0 +1,316 @@
+########################################################################################################################
+# Common variables
+########################################################################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The API Key to use for IBM Cloud."
+  sensitive   = true
+}
+
+variable "provider_visibility" {
+  description = "Set the visibility value for the IBM terraform provider. Supported values are `public`, `private`, `public-and-private`. [Learn more](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/guides/custom-service-endpoints)."
+  type        = string
+  default     = "private"
+
+  validation {
+    condition     = contains(["public", "private", "public-and-private"], var.provider_visibility)
+    error_message = "Invalid visibility option. Allowed values are 'public', 'private', or 'public-and-private'."
+  }
+}
+variable "existing_resource_group_name" {
+  type        = string
+  description = "Name of the existing resource group."
+}
+
+variable "region" {
+  type        = string
+  description = "The region to provision resources to."
+  default     = "us-south"
+}
+
+variable "prefix" {
+  type        = string
+  description = "The prefix to apply to all resources created by this solution."
```

Comment:
please update the description to match what is in https://github.ibm.com/GoldenEye/issues/issues/12841